src/main/java/DukeReadFile.java

```diff
@@ -0,0 +1,82 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Loads the tasks that were saved in a text document
+ * when the program starts up.
+ */
+public class DukeReadFile {
+
+	protected static String filePath;
+	protected static String currentTask = "";
+	protected static ArrayList<Task> ct = new ArrayList<>();
+	static String border = "____________________________________________________________";
```

Comment:
This is repeated from `Duke`. You should probably extract out a `Constants` class.